### ISSUE TITLE
Update to cdsw 1.0.1

### ISSUE THREAD

**Issue by TobyHFerguson** [TobyHFerguson/cdsw_install#5]
Modify repo to use baseurl of 1.0.1

```
[cloudera-cdsw]
# Packages for Cloudera's Distribution for data science workbench, Version 1, on RedHat	or CentOS 7 x86_64
name=Cloudera's Distribution for cdsw, Version 1
# old baseurl
# baseurl=https://archive.cloudera.com/cdsw/1/redhat/7/x86_64/cdsw/1/

baseurl=https://archive.cloudera.com/cdsw/1/redhat/7/x86_64/cdsw/1.0.1/
gpgkey =https://archive.cloudera.com/cdsw/1/redhat/7/x86_64/cdsw/RPM-GPG-KEY-cloudera    
gpgcheck = 1
```

probably

```sh
sed -i 's|/1/$|/1.0.1/' ....
```

**Comment by TobyHFerguson** [comment]
Need to ensure that the MASTER_IP_ADDRESS is an IP address, not a FQDN

**Comment by TobyHFerguson** [comment]
Fixed in 3be91e2c4
